web_widget_image_webcam/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=========================
+Web Widget - WebCam Image
+=========================
+
+This module was written to extend the functionality of the image widget and allows to take snap with WebCam.
+
+Configuration
+=============
+
+By default, the module is worked with every major browsers.
+
+An important note for Chrome 47+ users, this module only works with websites those have SSL / HTTPS.
+Visit this for `More Info
+<https://github.com/jhuckaby/webcamjs#important-note-for-chrome-47>`_.
+
+But, If you still want this module to work with websites those don't have SSL / HTTPS.
+Here is the steps to do it easily (Run in Adobe Flash fallback mode)
+
+* Go to file ...
+   **web_widget_image_webcam/static/src/js/webcam_widget.js**
+      * Set parameter **force_flash** to **true**          (line 34)
+      * Remove the comment tag form parameter **swfURL**   (line 39)
+
+Its done! Now this module also work with websites those don't have SSL / HTTPS.
+
```

Comment:
It would be nice if we have a quick Usage instructions as well. I was admittedly unclear on how to use the module until I looked at the instructions. 

Bravo though because it really is simple enough to not need them once you see the button.